resources/europe/france/fr-mastodon.json

```
@@ -0,0 +1,17 @@
+{
+  "id": "OSMFR-Mastodon",
```

```suggestion
  "id": "fr-mastodon",
```
Change id to 'fr' for countrywide resource and use lowercase, for consistency.